test/files/jvm/sync-var.check

```diff
@@ -1,2 +1,2 @@
-warning: there was one deprecation warning (since 2.13.0); re-run with -deprecation for details
+warning: there was 1 deprecation warning (since 2.13.0); re-run with -deprecation for details
```

Comment:
As a "by the way" extra requirement, could we change the compiler to write "there **were** 1 deprecation warning"?  When it comes to the compiler console output, It is more useful to help `grep` that appease English Grammar. 

> (ii) Expect the output of every program to become the input to another, as yet unknown, program.  (Doug McIlroy).

Reply:
That is the gist of the current change, to eliminate char sequences for both human and machine consumption. I will follow up by not emitting the extra words. Incredibly, "fixed in dotty":
```
$ dotc z.scala
there were 1 deprecation warning(s); re-run with -deprecation for details
```

Reply:
Also, why was it "was/were" instead of "is/are"? The compilation run was in the past, but the warnings are in the present.